store/views.py

```diff
@@ -77,6 +93,54 @@ def loanBookView(request):
 @csrf_exempt
 @login_required
 def returnBookView(request):
-    pass
+    book_id = request.POST.get("bid")
+    book = BookCopy.objects.get(pk=book_id)
+    if(book):
+      message ="success"
+      book.status = True
+      book.borrower = None
+      book.borrow_date = None
+      book.save()
+    else :
+      message = "failure" 
```

Comment:
There must be a validation in the backend when a user is returning the book, to make sure that he has only borrowed the book. Otherwise, a simple POST request will make the BookCopy to be returned, and would set its status as True.